### ISSUE TITLE
BUG - sev-3 - iOS - In iOS18, all device-phone-popups initially have focus pulled to a 'call 711' element that doesn't exist

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#9655]
## What happened?

When tapping any phone number in the app, in iOS, with the screen reader turned on - when the native device phone-related popup appears, the user's screenreader focus is pulled to an element (that doesn't exist visually) which announces "call 711":
<img width="374" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBemIyQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0ad252f207ec507bd60ab3812c324c24748dcbc6/image.png" alt="image.png" />

If you swipe to continue, you're brought to the "call [actual number you tapped on]" button of the native popup. At which point, if you attempt to swipe backwards, you get the 'there's nothing backwards/end of list" sound effect.

## Specs:

- Device:
- OS: iOS 18 (cannot reproduce on other iOS versions)
- User Account: any
- Accessibility State: VoiceOver on

## Steps to Reproduce

- With VoiceOver on, go to any page with a phone number (the Veterans Crisis Line is easy to access)
- Double-tap on any phone number
- Notice your screenreader focus is pulled to the non-existent "call 711" element
- Navigate through the rest of the popup just fine

## Desired behavior

Focus pulled directly to first button of native popup, as per all other iOS versions

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High Low 
- Frequency: High Low 

- 3 - Low

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))
